app/budget/services/budgetExcelService.js

```diff
@@ -0,0 +1,456 @@
+		data.push(row);
+		row = [];
+
+		// Total Student Credit Hours 
+		row.push("Student Credit Hours");
+		summary.terms.forEach(function(term) {
+			row.push(summary.byTerm[term].totalSCH);
+		});
+		row.push(summary.combinedTerms.totalSCH);
+		data.push(row);
+		row = [];
+
+		// Empty row
+		data.push([" "]);
+
+		// Lower Div Count
+		row.push("Lower Div Offerings");
+		summary.terms.forEach(function(term) {
+			row.push(summary.byTerm[term].lowerDivCount);
+		});
+		row.push(summary.combinedTerms.lowerDivCount);
+		data.push(row);
+		row = [];
+
+		// Upper Div Count 
+		row.push("Upper Div Offerings");
+		summary.terms.forEach(function(term) {
+			row.push(summary.byTerm[term].upperDivCount);
+		});
+		row.push(summary.combinedTerms.upperDivCount);
+		data.push(row);
+		row = [];
+
+		// Graduate Count
+		row.push("Graduate Offerings");
+		summary.terms.forEach(function(term) {
+			row.push(summary.byTerm[term].graduateCount);
+		});
+		row.push(summary.combinedTerms.graduateCount);
+		data.push(row);
+		row = [];
+
+		// Total Offerings
+		row.push("Total Offerings");
+		summary.terms.forEach(function(term) {
+			row.push(summary.byTerm[term].totalOfferingsCount);
+		});
+		row.push(summary.combinedTerms.totalOfferingsCount);
+		data.push(row);
+		row = [];
+
+		//Creates book
+		var wb = XLSX.utils.book_new(); // eslint-disable-line no-undef
+		// Creates Summary worksheet
+		var ws_summary = XLSX.utils.aoa_to_sheet(data); // eslint-disable-line no-undef
+
+		// Set column widths
+		var wscols = [
+			{wch: 20},
+			{wch: 15},
+			{wch: 15},
+			{wch: 15},
+			{wch: 15}
+		];
+		ws_summary['!cols'] = wscols;
+
+		/* add worksheet to workbook */
+		XLSX.utils.book_append_sheet(wb, ws_summary, 'Budget Summary'); // eslint-disable-line no-undef
+		// Cleans data for the next sheet
+		data.length = 0;
+
+
+// SHEDULE COST REPORT
+		// Header
+		data.push(['Term',
+								'Subject Code',
+								'Course Number',
+								'Title',
+								'Units High',
+								'Units Low',
+								'Sequence',
+								'Enrollment',
+								'Sections',
+								'Instructor',
+								'Regular Instructor',
+								'Reason',								
+								'TAs',
+								'Readers',
+								'TA Cost',
+								'Reader Cost',
+								'Support Cost',
+								'Instructor Cost',
+								'Total Cost']);
+
+		viewState.calculatedScheduleCosts.terms.forEach(function(term) {
+			var scheduleCosts = viewState.calculatedScheduleCosts.byTerm[term];
+			for (var i = 0; i < scheduleCosts.length; i++) {
+				var row = [];
+				let termDescription = termDescriptions[term];
+				row.push(termDescription);
+				row.push(scheduleCosts[i].subjectCode);
+				row.push(scheduleCosts[i].courseNumber);
+				row.push(scheduleCosts[i].title);
+				row.push(scheduleCosts[i].unitsHigh);
+				row.push(scheduleCosts[i].unitsLow);
+				
+				let sectionGroupCosts = scheduleCosts[i].sectionGroupCosts;
+				for (var _i = 0; _i < sectionGroupCosts.length; _i++) {
```

Comment:
Please refactor to follow code style conventions, our current code style has been to put a new line before for loops to separate them.